pkg/certmaker/templates/leaf-template.json

```diff
@@ -0,0 +1,28 @@
+{
+    "subject": {
+        "country": [
+            "US"
+        ],
+        "organization": [
+            "Sigstore"
+        ],
+        "organizationalUnit": [
+            "Fulcio Leaf CA"
+        ],
+        "commonName": "https://fulcio.com"
```

Comment:
Could we support templating (https://pkg.go.dev/text/template), or at least leave these values blank? I want to discourage copy-pasting of default values.

Reply:
i went ahead and made them blank for now / there's validation to ensure they're included, so the user will be prompted to fill them in if they forget to. 

maybe we could implement templating in another PR ?